src/crawlee/browsers/_base_browser_plugin.py

```diff
@@ -35,13 +35,13 @@ def browser_type(self) -> BrowserType:
 
     @property
     @abstractmethod
-    def browser_options(self) -> Mapping[str, Any]:
-        """Return the options for a new browser."""
+    def browser_launch_options(self) -> Mapping[str, Any]:
+        """Return the options for the `browser.launch` method."""
 
     @property
     @abstractmethod
-    def page_options(self) -> Mapping[str, Any]:
-        """Return the options for a new page."""
+    def browser_new_context_options(self) -> Mapping[str, Any]:
```

Comment:
Could we use the same docstring as in the PW crawler (except for the "This option should not be used if `browser_pool` is provided.")? This is also end-user interface.

Reply:
Done